template-helpers/hbs-helpers.js

```diff
@@ -50,56 +50,75 @@ function microsurveyBanner(args) {
     return;
   }
 
-  // don't show micro survey if language isn't English
-  // TODO: localize micro survey questions
-  if (!englishInAcceptLanguages(args)) {
-    return;
-  }
-
   // don't show micro survey if user is not signed in
   if (!args.data.root.req.session.user){
     return;
   }
 
-  const bannerOpeningDiv = "<div id=\"micro-survey-banner\" class=\"micro-survey-banner hidden\">";
+  const microSurveyResponseIds = [
+    "micro-survey-strongly-disagree-response",
+    "micro-survey-disagree-response",
+    "micro-survey-unsure-response",
+    "micro-survey-agree-response",
+    "micro-survey-strongly-agree-response",
+    "micro-survey-very-disappointed-response",
+    "micro-survey-somewhat-disappointed-response",
+    "micro-survey-dont-care-response",
+    "micro-survey-not-likely-response",
+    "micro-survey-very-likely-response",
+  ];
+  const enLocaleArgs = {
+    hash: args.hash, data: {root: {req: {supportedLocales: ["en"]} } },
+  };
+  let bannerOpeningDivDataset = "";
+  microSurveyResponseIds.forEach(id => {
+    bannerOpeningDivDataset += ` data-${id}-translated="${getString(id, args)}" `;
+    bannerOpeningDivDataset += ` data-${id}-english="${getString(id, enLocaleArgs)}" `;
+  });
+  const bannerOpeningDiv = `<div id="micro-survey-banner" class="micro-survey-banner hidden" ${bannerOpeningDivDataset}>`;
   const nowSecond = Math.abs(Math.floor(new Date().getTime() / 1000)) % 10;
```

Comment:
Q: Should this just be `new Date().getSeconds() % 10`?

Reply:
Updated